spec/jobs/outlook/update_event_job_spec.rb

```diff
@@ -0,0 +1,53 @@
+  let(:motif) { create(:motif, name: "Super Motif", location_type: :phone) }
+  # We need to create a fake agent to initialize a RDV as they have a validation on agents which prevents us to control the data in its AgentsRdv
+  let(:fake_agent) { create(:agent) }
+  let(:agent) { create(:agent, microsoft_graph_token: "token") }
+  let(:rdv) { create(:rdv, id: 20, motif: motif, organisation: organisation, starts_at: Time.zone.parse("2023-01-01 11h00"), duration_in_min: 30, agents: [fake_agent]) }
+  let(:agents_rdv) { create(:agents_rdv, id: 12, rdv: rdv, agent: agent, outlook_id: "super_id") }
+
+  context "when the event is updated" do
+    before do
+      stub_request(:patch, "https://graph.microsoft.com/v1.0/me/Events/super_id")
+        .with(
+          body: "{\"subject\":\"Super Motif\",\"body\":{\"contentType\":\"HTML\",\"content\":\"plus d'infos dans RDV Solidarités: http://www.rdv-solidarites-test.localhost/admin/organisations/10/rdvs/20\"},\"start\":{\"dateTime\":\"2023-01-01T11:00:00+01:00\",\"timeZone\":\"Europe/Paris\"},\"end\":{\"dateTime\":\"2023-01-01T11:30:00+01:00\",\"timeZone\":\"Europe/Paris\"},\"location\":{\"displayName\":\"Par téléphone\"}}",
+          headers: { "Accept" => "application/json", "Authorization" => "Bearer token", "Content-Type" => "application/json", "Expect" => "", "Return-Client-Request-Id" => "true",
+                     "User-Agent" => "RDVSolidarites", }
+        )
+        .to_return(status: 200, body: { id: "event_id" }.to_json, headers: {})
+
+      allow(Sentry).to receive(:capture_message)
+
+      described_class.perform_now(agents_rdv)
+    end
+
+    it "does not call Sentry" do
+      expect(Sentry).not_to have_received(:capture_message)
+    end
+  end
+
+  context "when the event cannot be updated" do
+    before do
+      stub_request(:patch, "https://graph.microsoft.com/v1.0/me/Events/super_id")
+        .with(
+          body: "{\"subject\":\"Super Motif\",\"body\":{\"contentType\":\"HTML\",\"content\":\"plus d'infos dans RDV Solidarités: http://www.rdv-solidarites-test.localhost/admin/organisations/10/rdvs/20\"},\"start\":{\"dateTime\":\"2023-01-01T11:00:00+01:00\",\"timeZone\":\"Europe/Paris\"},\"end\":{\"dateTime\":\"2023-01-01T11:30:00+01:00\",\"timeZone\":\"Europe/Paris\"},\"location\":{\"displayName\":\"Par téléphone\"}}",
+          headers: { "Accept" => "application/json", "Authorization" => "Bearer token", "Content-Type" => "application/json", "Expect" => "", "Return-Client-Request-Id" => "true",
+                     "User-Agent" => "RDVSolidarites", }
+        )
+        .to_return(status: 404, body: { error: { code: "TerribleError", message: "Quelle terrible erreur" } }.to_json, headers: {})
+
+      allow(Sentry).to receive(:capture_message)
+
+      described_class.perform_now(agents_rdv)
+    end
+
+    it "sends the error to Sentry" do
+      expect(Sentry).to have_received(:capture_message).with("Outlook API error for AgentsRdv 12: Quelle terrible erreur")
```

Comment:
Pareil que ci-dessus : 

```suggestion
      expect(sentry_events.last.message).to eq("Outlook API error for AgentsRdv 12: Quelle terrible erreur")
```